src/main/scala/sbtbuildinfo/ScalaCaseClassRenderer.scala

```diff
@@ -52,12 +53,12 @@ case class ScalaCaseClassRenderer(options: Seq[BuildInfoOption], pkg: String, ob
         .toList ::: List("")
     else Nil
 
-  private def caseClassDefinitionEnd = List(") {", "")
+  private def caseClassDefinitionEnd = List(s") $objTraits {", "")
   private def caseClassEnd = List("}")
 
   private def caseObjectLine(buildInfoResults: Seq[BuildInfoResult]) = List(
     s"case object $obj {",
-    s"  def apply(): $obj = new $obj(${buildInfoResults.map(_.value).map(quote).mkString(",")})",
+    s"  def apply(): $obj = new $obj(${buildInfoResults.map(r => s"\n    ${r.identifier} = ${quote(r.value)}").mkString(",")})",
```

Comment:
Could you explain why you want to switch to using named parameters here?

Reply:
I just see the generated code more clear than a whole line with the values.